packages/itwinui-react/src/core/Badge/Badge.tsx

```diff
@@ -17,22 +17,26 @@ import '@itwin/itwinui-css/css/badge.css';
  * Helper function that returns one of the preset badge color values.
  */
 const getBadgeColorValue = (color: BadgeProps['backgroundColor']) => {
+  const statuses = ['positive', 'negative', 'warning', 'informational'];
```

Comment:
I wasn't able to use `getStatusValue` here because the return type includes the status strings which aren't able to be passed into `isSoftBackground`. If there is a way around this, please let me know